Dockerfile

```
@@ -0,0 +1,36 @@
+ARG NODE_IMAGE_VERSION
+
+# Create base image
+FROM node:${NODE_IMAGE_VERSION} as base
+RUN apk add --no-cache tini python make g++
+
+# Install atlas engine
+FROM base as atlas_engine_admin_domain_minimal
+
+ARG ATLAS_ENGINE_ADMIN_DOMAIN_MINIMAL_VERSION
+
+COPY 'atlas_engine_admin_domain_minimal_linux.tar.gz' ./
+RUN tar zxf atlas_engine_admin_domain_minimal_linux.tar.gz
+RUN npm install sqlite3
+
+RUN npm link
+
+# Create release
+FROM atlas_engine_admin_domain_minimal as release
+EXPOSE 8400
```

```suggestion
EXPOSE 8200
```

Siehe: 
https://github.com/atlas-engine/AdminDomain.Application.Minimal/blob/develop/config/develop/http/http_extension.json#L4